### PR TITLE
Regenerate AWS API definitions automatically

### PR DESCRIPTION
**Pull request by mattBrzezinski**
## Overview
This merge request creates a GitHub action to regenerate AWS API definitions every day at 6a.m., if there are changes it will create a subsequent merge request with the changes.

This should be one of the last merge requests to go in.

### CODE REVIEW

File: .github/workflows/update_apis.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ on:
+  schedule:
+    - cron: '0 6 * * *'  # Daily at 6 a.m.
+
+jobs:
+  AWS:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        julia-version: [1.4.0]
```
Comment (nicoleepp) [inline .github/workflows/update_apis.yml:11]: Could we just use 

```suggestion
        julia-version: [1] 
```

to get the latest stable release?

File: .github/workflows/update_apis.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name: AWS
+ on:
+  schedule:
+    - cron: '0 6 * * *'  # Daily at 6 a.m.
```
Comment (mattBrzezinski) [inline .github/workflows/update_apis.yml:4]:
```suggestion
on:
  schedule:
    - cron: '0 6 * * *'  # Daily at 6 a.m.
```

File: .github/workflows/update_apis.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+name: AWS
+on:
+  schedule:
+    - cron: '0 6 * * *'  # Daily at 6 a.m.
```
Comment (omus) [inline .github/workflows/update_apis.yml:4]: Would be good to mention a time zone

Comment (mattBrzezinski) [inline .github/workflows/update_apis.yml:4]:
```suggestion
    - cron: '0 6 * * *'  # Daily at 6 a.m. UTC 
```

**Pull request merged**